### PR TITLE
feat(checks/windows/winbase): Add cpu count to windows perfos check

### PR DESCRIPTION
**Pull request by kaustavha**

### CODE REVIEW

**Comment by rphillips** [comment]
cc @rjemanuele look good to you? LGTM

**Comment by rjemanuele** [comment]
@rphillips @creationix @kaustavha  is this any better or different from https://github.com/virgo-agent-toolkit/rackspace-monitoring-agent/blob/master/check/cpu.lua#L62 ?

**Comment by kaustavha** [comment]
@rjemanuele yes its quite different, the l62 thing there returns a lot more data
Simpler example
https://github.com/virgo-agent-toolkit/rackspace-monitoring-agent/blob/master/hostinfo/cpu.lua
Then we need to actually count how many cpus it returned.
This (uv func in the commit) just returns a number.

**Comment by rjemanuele** [comment]
@jacob0719 is the cpu count the only metric item you need to improve this?

**Comment by jacob0719** [comment]
CPU count is all I need.

File: check/windows/winbase.lua [hunk 1]
```diff
@@ -148,6 +149,10 @@ function WindowsPowershellCmdletCheck:run(callback)
     child:on('error', function(err)
       checkResult:setStatus("err " .. err)
 
+      -- Get the cpu count
+      local vcpus = #uv.cpu_info()
+      checkResult:addMetric('cpu_count', nil, 'uint32', vcpus, '')
```
Comment (rphillips) [inline check/windows/winbase.lua]: we can leave out the last parameter... should be:

```
checkResult:addMetric('cpu_count', nil, 'uint32', vcpus)
```

+1 afterwards

Comment (rjemanuele) [inline check/windows/winbase.lua]: +1

**Comment by rjemanuele** [comment]
+1

**Comment by creationix** [comment]
Just so it's clear, the uv function returns more than just count too, but by counting the number of results, you can quickly get the count.

```lua
Welcome to the Luvit repl!
> require('uv').cpu_info()
{
  { times = { irq = 0, user = 44698590, idle = 839751040, sys = 21524210, nice = 0 },
    model = 'Intel(R) Core(TM) i5-4308U CPU @ 2.80GHz', speed = 2800 },
  { times = { irq = 0, user = 10602200, idle = 888886450, sys = 6476340, nice = 0 },
    model = 'Intel(R) Core(TM) i5-4308U CPU @ 2.80GHz', speed = 2800 },
  { times = { irq = 0, user = 42773400, idle = 846180740, sys = 17012570, nice = 0 },
    model = 'Intel(R) Core(TM) i5-4308U CPU @ 2.80GHz', speed = 2800 },
  { times = { irq = 0, user = 10937420, idle = 888572250, sys = 6455010, nice = 0 },
    model = 'Intel(R) Core(TM) i5-4308U CPU @ 2.80GHz', speed = 2800 } }
> #require('uv').cpu_info()
4
```

**Comment by kaustavha** [comment]
My bad, i misspoke.

File: check/windows/winbase.lua (hunk 1 above)
Comment (kaustavha) [inline check/windows/winbase.lua]: I fix. I merge now.

**Pull request merged**